modules/ROOT/pages/to-find-info.adoc

```diff
@@ -19,14 +19,20 @@ the HTTP functions in the REST specification.
 
 == Exchange Search
 
-Search lets you find Exchange assets that contain one or more terms
-in an asset title, asset ID, or a tag in an asset. You can also combine search terms with the filters listed in All Types.
-Searches only apply to the Exchange you are viewing.
-If you search in a private Exchange, the search only lists the
-private Exchange assets and not public Exchange assets.
+Exchange Search looks for terms in asset names, tags, categories, properties defined in API specifications, custom attributes (also known as custom fields), and the portal content shown on the asset page. Search includes RAML and OAS APIs and all other assets in Exchange.
+
+Assets migrated from Exchange 1 also allow the search to include text in an asset's description.
+
+You can combine search terms with the filters listed in All Types.
 
 You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
 
+Searches only apply to the Exchange you are viewing.
+
+If you search in a private Exchange, the search only lists the private Exchange assets and not public Exchange assets.
```

Comment:
private Exchange assets and ~~not~~ public Exchange assets? @wwarner-salesforce

Reply:
The current doc at https://docs.mulesoft.com/exchange/ says "Searches only apply to the Exchange you are viewing. If you search in a private Exchange, the search only lists the private Exchange assets and not public Exchange assets." Is this correct? It might have been an incorrect document, or one that was correct for a past version and is incorrect for the current version.

Reply:
This is ok

Reply:
I'd change that part in what you propose, @wwarner-salesforce "the search only lists the private Exchange assets and not public Exchange assets" for something like "the search only lists the private Exchange assets and not Mulesoft's public assets". Since you are getting the public assets of YOUR organization in this search

Reply:
Thank you very much! Confirmed with Mariano: "Searches only apply to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets."